### ISSUE TITLE
Revit: crash when running on german Revit

### ISSUE THREAD

**Issue by thoffeller** [specklesystems/speckle-sharp#542]
the connector crashes Revit when /language DEU is in use (maybe for other Revit languages, too) when iterating over the buildin categories and creating the list in ConnectorRevitUtils.cs, line 41.

Steps to reproduce:

- set revit to german with /language DEU
- load the fantastic rac_basic_sample_project.rvt
- open speckle connector
- click add stream

in german Revit it receives 2 times the same key for columns. Maybe a translation issue from Revit?

can be fixed like that:

```csharp
try
{
  var category = Category.GetCategory(doc, bic);
  _categories.Add(category.Name, category);
}
catch (Exception ex)
{
}
```

**Comment by thoffeller** [comment]
sorry, I'm not allowed to add labels ?

**Comment by AlanRynne** [comment]
Hey @thoffeller, thanks for reporting this! We'll have a look at it 👍🏻  I added the tags for you already ;)

**Comment by teocomi** [comment]
Hey @thoffeller I've pushed a hotfix for this and triggered a beta release of the Revit connector, otherwise it'll be included in the next stable release in ~ 1.5 weeks.
Hopefully no more unexpected crashes :)
